podcasts/Referrals/Banner/ReferralsClaimBannerTableCell.swift

```diff
@@ -0,0 +1,30 @@
+import UIKit
+import SwiftUI
+
+class ReferralsClaimBannerTableCell: ThemeableCell {
+    static var identifier = "ReferralsClaimBannerTableCellIdentifier"
+
+    override init(style: UITableViewCell.CellStyle, reuseIdentifier: String?) {
+        super.init(style: style, reuseIdentifier: reuseIdentifier)
+        let viewModel = ReferralClaimPassModel()
+
+        let bannerView = ReferralsClaimBannerView(viewModel: viewModel).themedUIView
+        bannerView.backgroundColor = .clear
+        contentView.addSubview(bannerView)
+
+        bannerView.translatesAutoresizingMaskIntoConstraints = false
+        NSLayoutConstraint.activate([
+            bannerView.leadingAnchor.constraint(equalTo: contentView.leadingAnchor, constant: 16.0),
+            bannerView.trailingAnchor.constraint(equalTo: contentView.trailingAnchor, constant: -16.0),
```

Comment:
Could these `16` values go in a Constants Enum for horizontal spacing?

Reply:
Done here: https://github.com/Automattic/pocket-casts-ios/pull/2158/commits/2f6f95b56e195054af066036208a783096f89ef0